src/Chapter1.hs

```diff
@@ -520,7 +523,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if (abs x) < abs(y) then x else y
```

Comment:
In Haskell, you don't need parenthesis in both of these cases:

```suggestion
closestToZero x y = if abs x < abs y then x else y
```

Also, functions and arguments are always separated with spaces even when arguments are in parenthesis 👾